### ISSUE TITLE
All mail's from different domains are marked as an spam in Microsoft's provided email service

### ISSUE THREAD

**Issue by Pka78** [mailcow/mailcow-dockerized#1371]
Hello.
I have quite many, 5 -domains that i'll use to send email. There is 4 nonprofit organizations and 1 of my own. All messages that i send trough Mailcow are marked as an spam in Microsoft's services. Marking is done by SmartScreen filter, so i'm kinda lost with this one. Microsoft support service wrote that e-mail headers should follow "best practises" but i'm not sure what best practises are men't at this point. 
It doesn't matter that there is or isn't SPF and DKIM keys in DNS & message headers. My DNS Configuration are correct with every test service that i have taken. 

This is only when i send mail trough my MailCow server. 

Message header & message source:

Received: from SN1NAM01HT135.eop-nam01.prod.protection.outlook.com (2603:10a6:209:15::14) by AM4PR08MB0769.eurprd08.prod.outlook.com with HTTPS via AM6PR0202CA0001.EURPRD02.PROD.OUTLOOK.COM; Sat, 5 May 2018 06:26:25 +0000 Received: from SN1NAM01FT059.eop-nam01.prod.protection.outlook.com (10.152.64.51) by SN1NAM01HT135.eop-nam01.prod.protection.outlook.com (10.152.65.188) with Microsoft SMTP Server (version=TLS1_2, cipher=TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA384) id 15.20.715.13; Sat, 5 May 2018 06:26:24 +0000 Authentication-Results: spf=pass (sender IP is 193.70.39.8) smtp.mailfrom=kajander.info; outlook.com; dkim=pass (signature was verified) header.d=kajander.info;outlook.com; dmarc=pass action=none header.from=kajander.info; Received-SPF: Pass (protection.outlook.com: domain of kajander.info designates 193.70.39.8 as permitted sender) receiver=protection.outlook.com; client-ip=193.70.39.8; helo=mailhost.kajander.info; Received: from mailhost.kajander.info (193.70.39.8) by SN1NAM01FT059.mail.protection.outlook.com (10.152.65.115) with Microsoft SMTP Server (version=TLS1_2, cipher=TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA384) id 15.20.715.13 via Frontend Transport; Sat, 5 May 2018 06:26:23 +0000 X-IncomingTopHeaderMarker: OriginalChecksum:C365FCFA734A6DEF3D6EB84A68E0DB4622C0066695140A5FFBAFDA9DF19C06C6;UpperCasedChecksum:235C6C0986C9AA5E106AB5FDEFD9EE915E9358FD32E98BCA604B33C88371EB3A;SizeAsReceived:2298;Count:15 Received: from 7c0fd1eab4c5 (mailcowdockerized_sogo-mailcow_1.mailcowdockerized_mailcow-network [172.22.1.248]) (Authenticated sender: petri@kajander.info) by mailhost.kajander.info (Postfix) with ESMTPA id 872543F1B4 for <kuva.kajander@outlook.com>; Sat, 5 May 2018 09:25:34 +0300 (EEST) DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=kajander.info; s=mailhost; t=1525501534; h=from:sender:reply-to:subject:date:message-id:to:cc:mime-version:content-type:content-transfer-encoding:resent-to:resent-cc:resent-from:resent-sender:resent-message-id:in-reply-to:references:list-id:list-owner:list-unsubscribe:list-subscribe:list-post; bh=oSoGVI3P2ILST0/R7k3f+sfuW1s4l2aLen9jMYuEF3E=; b=PC9y+hU4w5gqmETCWlXuaq0lm/Q2teU6w/zwozVsw62dEWz4sVOnfojO62c01shhQmKwwq 69jcARwIdadNTY7bZNWrLeIOK5Le/cnZmnSEfrXZvDzeYJqNPn8f77H8yMXZV6pc8dUIv2 Xe7uWseMdKoGfta4xNC1fTpT/0wqpEo= From: "Petri Kajander" <petri@kajander.info> To: <kuva.kajander@outlook.com> Subject: =?iso-8859-1?Q?Testis=E4hk=F6posti?= Date: Sat, 5 May 2018 09:26:20 +0300 Message-ID: <008001d3e439$fb8b5e40$f2a21ac0$@kajander.info> Content-Type: multipart/alternative; boundary="----=_NextPart_000_0081_01D3E453.20D93280" X-Mailer: Microsoft Outlook 16.0 Thread-Index: AdPkOfU7tHEYj4HySjaVVHZ2hhuWdg== Content-Language: fi ARC-Message-Signature: i=1; a=rsa-sha256; c=relaxed/relaxed; d=kajander.info; s=mailhost; t=1525501534; h=from:sender:reply-to:subject:date:message-id:to:cc:mime-version:content-type:content-transfer-encoding:resent-to:resent-cc:resent-from:resent-sender:resent-message-id:in-reply-to:references:list-id:list-owner:list-unsubscribe:list-subscribe:list-post; bh=oSoGVI3P2ILST0/R7k3f+sfuW1s4l2aLen9jMYuEF3E=; b=N6xln0sVSNexo8fx8AAb5KlQBRsnPtVK+m9gxzO7NBkhdoNRCbBP+VcQ1EJBNmi6dBzMrs o8swnoM5b9llotnceMDB3sOJ9V1fh3S8DSvv6cmu30ym+/zrPT0wLx9s/A7WBpaRoYENat baRkAvJSGcDf0lYyLH1U6VAov8ytUmI= ARC-Seal: i=1; s=mailhost; d=kajander.info; t=1525501534; a=rsa-sha256; cv=none; b=nUdKM9CozmcpQStWSetHazCugfnCc/4iq+uI2HF0IoidWAs5dMiEw0H2ftNAwks9qA9R+pVZbKtHiYrTc4EVROCRcvP9qkJ8AZIpuqbq3bzulbsUSbDfWaT5zY08jJuJWcP17j6uMmfkOjrL22pTIklxu6rqtHhuml43uCFcp24= ARC-Authentication-Results: i=1; auth=pass smtp.auth=petri@kajander.info smtp.mailfrom=petri@kajander.info X-IncomingHeaderCount: 15 Return-Path: petri@kajander.info X-MS-Exchange-Organization-ExpirationStartTime: 05 May 2018 06:26:23.8932 (UTC) X-MS-Exchange-Organization-ExpirationStartTimeReason: Original Submit X-MS-Exchange-Organization-ExpirationInterval: 2:00:00:00.0000000 X-MS-Exchange-Organization-ExpirationIntervalReason: Original Submit X-MS-Exchange-Organization-Network-Message-Id: efb0a4e4-5d99-4350-df0b-08d5b2511fd4 X-EOPAttributedMessage: 0 X-EOPTenantAttributedMessage: 84df9e7f-e9f6-40af-b435-aaaaaaaaaaaa:0 X-MS-Exchange-Organization-MessageDirectionality: Incoming X-Microsoft-Exchange-Diagnostics: 1;SN1NAM01FT059;1:AdSDX+kNWK1hFokbhzN0srb9FXrBkzApD9Zp8wBvBEFUqVMMb3D6DC0npveT5FmlycUFDjbdulwBhGedR59/hu5gRvTjeWM1LwJ/EOoaD8lj+bUQsU6uMcTwvyFHbH7/ X-Forefront-Antispam-Report: EFV:NLI; X-MS-Exchange-Organization-AuthSource: SN1NAM01FT059.eop-nam01.prod.protection.outlook.com X-MS-Exchange-Organization-AuthAs: Anonymous X-MS-PublicTrafficType: Email X-Microsoft-Antispam: BCL:0;PCL:0;RULEID:(5000109)(4605076)(610169)(650170)(651021)(8291501071);SRVR:SN1NAM01HT135; X-Microsoft-Exchange-Diagnostics: 1;SN1NAM01HT135;3:sTcY2XUEmV9mM5oaFlJJiv/1yrC+ZhxTMbawbSSARXS5IW7JIgul6U1zTPlnrGI4fyQDxbN05UYI6VNmPCVvjnBKUqXMPx5/wnGK1QWwS7NfZODLbxOrla/5MoEU0d9uFlzRexq6erE8NIqmAd4e5sdqzkv7FZXkfwZE7aranBP9XhNWkzloyJmRxpDr4z/l2W6+MisGI8Gmyk3XbtyNJGxJwx1mHuxQj4Jg+PzJ8e89Rvo5Gk9HtbfgcoxDMsuZZjb0u7Gfe0Ouf69ySft5fgQbSEVxept03JKBR6WRj1+d9ZjmKtvR7xQKRKzjjcYO;25:1NYcUI5kLEi8dUHDcBlT9pPnwxZN6wpuSODYGkq5ljMtB+OBGYyczAbgYE9Cwtwgy0qT9z3Id87L8iGVsccKKpLG8PCAKeReB8r0OQDr9myLFvZzSkbOSLSa6v0uLwmHpeT01gIwNH/jn2QxHmOfv/lNFikqUH10kM1vK3b56J1mhrwNQaKxSr4bF5mE1r/Knff4rtqyHeZYKYLR7kQ5hcEXUvZOHlg1W1bsp2O6rZZZEl/+yjhcarkv+Vxb5avVTYHxG6c27Vp4oIUcYAAr5/nTR9UY3FoO8Esi2zXxabm6XNmEOVxfBTop50uVmnOPwnIhAyp6nA9HfXMIKcRPkg==;31:o1cqUAJsCtYjBBErRu3dh6Ad4+fs+KXHUNyJLfPvkjeotFpt6lFfcfk/6ncYhslBoSBUwc2ve4PqQ6rfULcN5OsgF/SapuW5knQYk3fPz4ZcF1GU+JQyFWcYQ+fCtXNS5dyqDp4nonOn2aWMVJSzrSUYkrRQneHv/H3Qpb32onCD6NC1N+7k2EiKuoDox2o37drsEp2tDXeN3IRhdwegRJGgum255ouKhumVyK+wFmY= X-MS-TrafficTypeDiagnostic: SN1NAM01HT135: X-MS-Exchange-EOPDirect: true X-Sender-IP: 193.70.39.8 X-SID-PRA: PETRI@KAJANDER.INFO X-SID-Result: PASS X-MS-Exchange-Organization-PCL: 2 X-Exchange-Antispam-Report-Test: UriScan:(28532068793085)(21748063052155); X-Exchange-Antispam-Report-CFA-Test: BCL:0;PCL:0;RULEID:(444111676)(52401190)(52601095)(52505095)(52406095)(82015058);SRVR:SN1NAM01HT135;BCL:0;PCL:0;RULEID:;SRVR:SN1NAM01HT135; X-Microsoft-Exchange-Diagnostics: 1;SN1NAM01HT135;4:eMYTxH3xqSkFkh4BhTCZzG1IuwPrbgt1wGyEa4UvLgH4yt/RfEw1hqg/rqLmoiCKuP6b5/SinTQ9rdcD+mmjcwoyVVI+uZDzjcw1X0w3GqgNJ4jZl96qLmC1mBI/DqjafWcT3K6U6iMQUPpCKqAmDFeuAG9+grCqbSjzHNKRMLC14CQCblW5xPAUtMqWyn74jvoKnT3736cUHvRagGvxmhQQ4JOOkljnkVYkb8fb3zY+qoL/3nj2XyFCVaK3WhNHcF7FVxFpHOlPVZMX2vdUf92mxZjTHrXCZEFuAAp0fIbL0e7FAJ1qhcrS/eDfHBW501q6bR/SC1C1EsAK1zW+L49eUtWRGODXjpbriXSynNs=;6:rz/NerYJl4uKe+NFvKGi9Q7meaGdrMguykZDYBicghQKaDK6bv2lfO+0l2S/bvEBd3s/hUp6E2fjsp1IvnJUFVSOKwy4ZguDoZB22dPiGF3qgKSRxkUAQQPrqghFfaLRMZGhTa5ZWlRdV8Kangv3CjHwrkjZw9CwxVGMuzkFbzxtcxOBTlYv3h+KuuHiIWFx8Xwh+6oM7lPYR/DRxEaA9zP8nteQ0uv2i1bmP7xyAM9Dgcue0AS9lYHKQb1NhT9H8YjqbwsgOOOsEK56XWj5AKw967aRrolembkvkb0iZ7VaRjxI3wekE3OfIVXLe6FHkNDwY5ISB0ZR7vuUXlA+z2sqyMDQuBY5ofOLSdGx8faM12/S5x2Kc0hfovN43Ms/FY6vHUHLJjEJHWAvG3RGqKhRVQkYN0r3PNKvd788skxKK4ApDynmDZuVxpUKTKp1vuAixK2dnNAbXrHVZnmjDw== X-Microsoft-Antispam-Message-Info: 7Z2afotbqyO7Rk7be+WVuYiv6hH+lYqkySWHU0d0k3rOTUVGyR4EwY6M9fYxinfDw0HgDa0DqYB1qz4v0hO3Nc/I3s2FQM+O4sHYP4UQsNmZKl6RoctnGihgMZqKv0z78Fl9RQ0bDdYES1o/9aJc6zawmEw5LKqBl6Egg5PbikRvVMjGPxyblMIg7Ga3r4Ev X-Microsoft-Exchange-Diagnostics: 1;SN1NAM01HT135;5:71tuzwKPWIs6yYzhK+ktH0bhmbB8LRv/oDR1sprbJerzb/yVhp+/KHagsO/x+5IHkDCubPdukflFtgk4dJ1qJC49Q4GqOUc1OZZiB/CgeEFzLopO+Gs2Z5HoeMHBxhxeMNN1TjVqtoTZIAK+nyofR1dW8bWdwDTWb6Tn8zkYaeQ=;7:9Dhip7IR1jB1ZqMxOLucuyxU1Z+79AUZ1Nc1ze+gpkL2KcziJvwc+Tsz6HHV+ohc25Lw3dcBZNJt8j9Yoepg43fa67OgYX1ZzTcC1IlqKYRNzpR5AcZQ8L8eiR5z64YK86JOGOUcBYKjaqtEgaOvaDe1glLvCEsPc7s+OJq3kEW93RxBzbwiIWmce/gx9p6fPaToVMVYHrwQmBe+hknEE1KFKU8Skgo5pIE6bYrcekd9pEzZx4ybBM3VrtIwagwx X-MS-Office365-Filtering-Correlation-Id: efb0a4e4-5d99-4350-df0b-08d5b2511fd4 X-OriginatorOrg: outlook.com X-MS-Exchange-CrossTenant-OriginalArrivalTime: 05 May 2018 06:26:23.7370 (UTC) X-MS-Exchange-CrossTenant-Network-Message-Id: efb0a4e4-5d99-4350-df0b-08d5b2511fd4 X-MS-Exchange-CrossTenant-Id: 84df9e7f-e9f6-40af-b435-aaaaaaaaaaaa X-MS-Exchange-CrossTenant-FromEntityHeader: Internet X-MS-Exchange-CrossTenant-RMS-PersistedConsumerOrg: 54485d23-c432-40fe-8436-6091d627118c X-MS-Exchange-Transport-CrossTenantHeadersStamped: SN1NAM01HT135 X-MS-Exchange-Transport-EndToEndLatency: 00:00:01.6421070 X-MS-Exchange-Processed-By-BccFoldering: 15.20.0735.000 X-Microsoft-Exchange-Diagnostics: 1;AM4PR08MB0769;27:+qZ+H+q7RKF7C1iULV6pdJ+PHygMmyfxnu9L6EByrLlDiGFemlOoCvxOw9MPBE8HrKAZZrxnKrf1kTqxBnk+JSgT4j1dfP3qBh616TJ7A7jze4ZKOsByF27rrk6+pXyK X-Microsoft-Antispam-Mailbox-Delivery: abwl:0;wl:0;pcwl:0;kl:0;iwl:0;dwl:0;dkl:0;rwl:0;ex:0;auth:1;dest:J;ENG:(400001000128)(400125000095)(5062000261)(5061607266)(5061608174)(4900095)(4920089)(6375004)(4950130)(4990090)(9140004);RF:JunkEmail;OFR:SpamFilterAuthJ; X-Message-Info: qoGN4b5S4yo8QlmcMYoX/JZ4KVuPs6lpx2edZ8jY0gVymzMZcUatsVv6ST8q/iOYJwrFFcy/JQpyEOU9qrJpC7xsZz2lBFa9i6LtGd9QNnBCeNUPkrNoJfSt1+z04Pe3c2vq8ddSlGPQi+S+PetqlonBiiKb4qzK9WQJ4GCNINFTBDy9+l2bwtz/ZDOJ836BnYots/o12aA1oVJ9rv29GQ== X-Message-Delivery: Vj0xLjE7dXM9MDtsPTA7YT0wO0Q9MjtHRD0yO1NDTD02 X-Microsoft-Antispam-Message-Info: 5rO84hhKtXzUVwSsNlv22D+jrtBx/6CtrAb2yOZz3OBYJSWKVLM8v4sCp+7Rl4XEYMQtMFiEmuKWYXyRxkdDshrY4ocXwYJ7+NszAucoV7Km2z3RVLmZNJgPoRsQfemCT5KLg4PDXXh1O9iH7IhgJ+xO4ysHeTy7MunfXsvk10l4f5hTNSctUPiMx7JHyjw0TYOK+OhVXg6GLMK4hTj0gSEqXHCYbw3KwsexrWMFY1VSLfs/67mW5h5IrtyMpdHFvCE8eFgXWmdunePPvVJTx+I6I4l7vmYxRoDld1KS1/y9EgsY0lZZhTREW5sZTcistapTeBD/sbN8qaokkh4Vh/NZWn4ecbrnoXykOzEybro= MIME-Version: 1.0 ------=_NextPart_000_0081_01D3E453.20D93280 Content-Type: text/plain; charset="iso-8859-1" Content-Transfer-Encoding: quoted-printable X-Microsoft-Exchange-Diagnostics: 1;AM4PR08MB0769;27:+qZ+H+q7RKF7C1iULV6pdJ+PHygMmyfxnu9L6EByrLlDiGFemlOoCvxOw9MPBE8HrKAZZrxnKrf1kTqxBnk+JSgT4j1dfP3qBh616TJ7A7jze4ZKOsByF27rrk6+pXyK X-Microsoft-Antispam-Mailbox-Delivery: abwl:0;wl:0;pcwl:0;kl:0;iwl:0;dwl:0;dkl:0;rwl:0;ex:0;auth:1;dest:J;ENG:(400001000128)(400125000095)(5062000261)(5061607266)(5061608174)(4900095)(4920089)(6375004)(4950130)(4990090)(9140004);RF:JunkEmail;OFR:SpamFilterAuthJ; X-Message-Info: qoGN4b5S4yo8QlmcMYoX/JZ4KVuPs6lpx2edZ8jY0gVymzMZcUatsVv6ST8q/iOYJwrFFcy/JQpyEOU9qrJpC7xsZz2lBFa9i6LtGd9QNnBCeNUPkrNoJfSt1+z04Pe3c2vq8ddSlGPQi+S+PetqlonBiiKb4qzK9WQJ4GCNINFTBDy9+l2bwtz/ZDOJ836BnYots/o12aA1oVJ9rv29GQ== X-Message-Delivery: Vj0xLjE7dXM9MDtsPTA7YT0wO0Q9MjtHRD0yO1NDTD02 X-Microsoft-Antispam-Message-Info: 5rO84hhKtXzUVwSsNlv22D+jrtBx/6CtrAb2yOZz3OBYJSWKVLM8v4sCp+7Rl4XEYMQtMFiEmuKWYXyRxkdDshrY4ocXwYJ7+NszAucoV7Km2z3RVLmZNJgPoRsQfemCT5KLg4PDXXh1O9iH7IhgJ+xO4ysHeTy7MunfXsvk10l4f5hTNSctUPiMx7JHyjw0TYOK+OhVXg6GLMK4hTj0gSEqXHCYbw3KwsexrWMFY1VSLfs/67mW5h5IrtyMpdHFvCE8eFgXWmdunePPvVJTx+I6I4l7vmYxRoDld1KS1/y9EgsY0lZZhTREW5sZTcistapTeBD/sbN8qaokkh4Vh/NZWn4ecbrnoXykOzEybro= Oisko t=E4m=E4 jo muuttunut? =20 //PKa ------=_NextPart_000_0081_01D3E453.20D93280 Content-Type: text/html; charset="iso-8859-1" Content-Transfer-Encoding: quoted-printable X-Microsoft-Exchange-Diagnostics: 1;AM4PR08MB0769;27:+qZ+H+q7RKF7C1iULV6pdJ+PHygMmyfxnu9L6EByrLlDiGFemlOoCvxOw9MPBE8HrKAZZrxnKrf1kTqxBnk+JSgT4j1dfP3qBh616TJ7A7jze4ZKOsByF27rrk6+pXyK X-Microsoft-Antispam-Mailbox-Delivery: abwl:0;wl:0;pcwl:0;kl:0;iwl:0;dwl:0;dkl:0;rwl:0;ex:0;auth:1;dest:J;ENG:(400001000128)(400125000095)(5062000261)(5061607266)(5061608174)(4900095)(4920089)(6375004)(4950130)(4990090)(9140004);RF:JunkEmail;OFR:SpamFilterAuthJ; X-Message-Info: qoGN4b5S4yo8QlmcMYoX/JZ4KVuPs6lpx2edZ8jY0gVymzMZcUatsVv6ST8q/iOYJwrFFcy/JQpyEOU9qrJpC7xsZz2lBFa9i6LtGd9QNnBCeNUPkrNoJfSt1+z04Pe3c2vq8ddSlGPQi+S+PetqlonBiiKb4qzK9WQJ4GCNINFTBDy9+l2bwtz/ZDOJ836BnYots/o12aA1oVJ9rv29GQ== X-Message-Delivery: Vj0xLjE7dXM9MDtsPTA7YT0wO0Q9MjtHRD0yO1NDTD02 X-Microsoft-Antispam-Message-Info: 5rO84hhKtXzUVwSsNlv22D+jrtBx/6CtrAb2yOZz3OBYJSWKVLM8v4sCp+7Rl4XEYMQtMFiEmuKWYXyRxkdDshrY4ocXwYJ7+NszAucoV7Km2z3RVLmZNJgPoRsQfemCT5KLg4PDXXh1O9iH7IhgJ+xO4ysHeTy7MunfXsvk10l4f5hTNSctUPiMx7JHyjw0TYOK+OhVXg6GLMK4hTj0gSEqXHCYbw3KwsexrWMFY1VSLfs/67mW5h5IrtyMpdHFvCE8eFgXWmdunePPvVJTx+I6I4l7vmYxRoDld1KS1/y9EgsY0lZZhTREW5sZTcistapTeBD/sbN8qaokkh4Vh/NZWn4ecbrnoXykOzEybro= <html xmlns:v=3D"urn:schemas-microsoft-com:vml" xmlns:o=3D"urn:schemas-micr= osoft-com:office:office" xmlns:w=3D"urn:schemas-microsoft-com:office:word" = xmlns:m=3D"http://schemas.microsoft.com/office/2004/12/omml" xmlns=3D"http:= //www.w3.org/TR/REC-html40"><head> <meta http-equiv=3D"Content-Type" content=3D"text/html; charset=3Diso-8859-= 1"><meta name=3D"Generator" content=3D"Microsoft Word 15 (filtered medium)"= ><style></style></head><body lang=3D"FI" link=3D"#0563C1"= vlink=3D"#954F72"><div class=3D"WordSection1"><p class=3D"MsoNormal">Oisko= t=E4m=E4 jo muuttunut?<o:p></o:p></p><p class=3D"MsoNormal"><o:p>&nbsp;</o= :p></p><p class=3D"MsoNormal">//PKa<o:p></o:p></p></div></body></html>= ------=_NextPart_000_0081_01D3E453.20D93280--

**Comment by andryyy** [comment]
Hi,

I’m on mobile and couldn’t check: is it OVH or some other ISP that doesn’t care much about their IP reputation? Most OVH IP ranges are almost useless  for mailing.

**Comment by tkorves** [comment]
Just checked: Yes, it's OVH.

route:          193.70.0.0/17
descr:          OVH
origin:         AS16276
mnt-by:         OVH-MNT
created:        2016-10-07T08:51:27Z
last-modified:  2016-10-07T08:51:27Z
source:         RIPE

**Comment by Braintelligence** [comment]
@andryyy Up till now, for every OVH IP address I had to verify my IP range to 2-3 providers first, who blacklist for no reason, but then I had no problems afterwards. Which server provider would you recommend instead?

**Comment by tkorves** [comment]
I'd suggest Servercow - run by @andryyy ...

**Comment by Pka78** [comment]
Yes, it's in OVH but according to MS it is not the servers reputation. According to every single test i have made it's in reputation 50/50 (Neutral). 

It's that god damn Smartscreen filter 

"My name is Punith and I work with the Outlook.com Deliverability Support Team.
We have reviewed your IP (193.70.39.8) and determined that messages are being filtered (i.e. sent to the Junk folder) based on the recommendations of the SmartScreen® Filter.
Email filtering is based on many factors, but primarily it's due to mail content and recipient interaction with that mail.  Because of the proprietary nature of SmartScreen® and because SmartScreen® Filter technology is always adapting and learning more about what is and isn't unwanted mail, it is not possible for us to offer specific advice about improving your mail content. However, in general SmartScreen® Filter evaluates specific words or characteristics from each e-mail message and weights them, based on their likelihood to indicate that a message is unwanted or legitimate mail. 
Unfortunately, after reviewing the information you provided and in compliance with our mail policies, we are unable to offer immediate mitigation for your deliverability issue. However, we have some specific recommendations for you to consider that can help you to improve deliverability over time.
Here are some specific recommendations for you to consider:
Brand your mail
Ensure mails are cleanly formatted and clearly identifiable as originating from your service. "

Servercow, in German only?

**Comment by Braintelligence** [comment]
@Pka78  Follow the steps here: https://wiki.hetzner.de/index.php/Microsoft_Blacklist#JMRP Especially concerning Delisting and JMRP.

I never had any problems after that.

EDIT: Sorry, mentioned wrong guy.

**Comment by tkorves** [comment]
I don't need to follow this ... ;-) I've got no problems at all ... ;-) Maybe @Pka78 should follow this.

**Comment by Pka78** [comment]
tkorves. Tried that one, they said that it's smartscreen and they can't do anything about it.

**Comment by tkorves** [comment]
@Pka78 I'd suspect it's not *only* Smartscreen - I guess it's a mixture of Smartscreen and IP reputation. And yes, OVH may or may not be the best choice for a mailserver - everyone has to decide it for themselves.

**Comment by Braintelligence** [comment]
For one of my IP addresses they answered with some gibberish, but after their answer everything just worked.
Are you sure you tried http://go.microsoft.com/fwlink/?LinkID=614866&clcid ?

**Comment by andryyy** [comment]
You will never be happy with AS16276. I think every other AS is better than AS16276. 👍

**Comment by discofris** [comment]
Very satisfied with AS63949 :-)

**Comment by Mupohkopmec** [comment]
Hi!
Well, after all this info above, is there something simple I can do to stop
whatsoever is happening with my email accounts and my internet activity
deviation and privacy invasion?
Thank you in advance.

Best regards,
Mupoh

Em qua, 9 de mai de 2018 13:43, André Peters <notifications@github.com>
escreveu:

**Comment by scher200** [comment]
@Braintelligence thank you for this link, at first it did not help. After subscribing the form it auto redirected me to: [https://support.microsoft.com/en-us/getsupport?oaspworkflow=start_1.0.0.0&wfname=capsub&productkey=edfsmsbl3&locale=en-us&forceorigin=esmc](https://support.microsoft.com/en-us/getsupport?oaspworkflow=start_1.0.0.0&wfname=capsub&productkey=edfsmsbl3&locale=en-us&forceorigin=esmc) They send me a mail with: "You are not receiving mitigation for your server", but according to this mail I was able to reply. So I did, describing them my story and scores on https://mxtoolbox.com/, it took a while but after some days they confirmed to me that now I was applied for mitigation and my server started working again. I have to add that I also did make sure my PTR record (reverse DNS record) was set and equal to mail.mydomain.com. This last thing was very clear to me while reading the mailcow DNS manual.
